Sources/Database/Database.swift

```diff
@@ -498,21 +499,24 @@ private struct _Client {
   // TODO: This should return a non-optional user
   func upsertUser(
     withGitHubEnvelope envelope: GitHubUserEnvelope,
-    email: EmailAddress
+    email: EmailAddress,
+    now: () -> Date
     ) -> EitherIO<Error, Models.User?> {
 
     return self.execute(
       """
-    INSERT INTO "users" ("email", "github_user_id", "github_access_token", "name", "episode_credit_count")
-    VALUES ($1, $2, $3, $4, 1)
+    INSERT INTO "users"
+    ("email", "github_user_id", "github_access_token", "name", "episode_credit_count")
+    VALUES ($1, $2, $3, $4, $5)
     ON CONFLICT ("github_user_id") DO UPDATE
     SET "github_access_token" = $3, "name" = $4
     """,
       [
         email.rawValue,
         envelope.gitHubUser.id.rawValue,
         envelope.accessToken.accessToken,
-        envelope.gitHubUser.name
+        envelope.gitHubUser.name,
+        envelope.gitHubUser.createdAt.distance(to: now()) < 60*60*24*7 ? 0 : 1
```

Comment:
Limiting to accounts older than a week for now...could tweak this value, though.

Reply:
Jj